client/client-lib/src/mint/backup.rs

```diff
@@ -321,6 +326,49 @@ impl MintClient {
     }
 }
 
+/// Backup metadata
+///
+/// A backup can have a blob of extra data encoded in it.
+/// We provide methods to use json encoding, but in clients
+/// are free to use their own encoding.
```

Comment:
nit :)

/// A backup can have a blob of extra data encoded in it.
/// We provide methods to use JSON encoding, but ~~in~~ clients
/// are free to use their own encoding.